recipes/unlzw3/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "unlzw3" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unlzw3-{{ version }}.tar.gz
+  sha256: eaf26c213100d886cea513e2cd77cc2fe418192789197c685259aa070fb88829
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - unlzw3
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/scivision/unlzw3
+  summary: Pure Python decompression module for .Z files compressed using Unix compress utility
+  license: MIT
```

Despite what metadata is set in `setup.cfg`, the text (which is what matters) is the zlib license:
```suggestion
  license: Zlib
```